doc/users/explain/figures.rst

```diff
@@ -0,0 +1,188 @@
+
+.. _figure_explanation:
+
+================================================
+Creating, viewing, and saving Matplotlib Figures
+================================================
+
```

Comment:
We should define what a figure is. Maybe something like

> In matplotlib, a `~.Figure` is the logical container for all visual elements. Think of it like a blank sheet of paper.
> To add content, you'll add visual elements (called `.Artist`\s) to the figure though specific functions.

This is somewhat informal, but should give a rough mental model.

Reply:
I think thats what the first two sentences below the example were meant to do?

Reply:
> When looking at Matplotlib visualization, you are almost always looking at
Artists placed on a `~.Figure`. 

Has too little emphasis on *Figure*. *Matplotlib visualization* and *Artists* are mentioned first and are the syntactic objects in the sentence. *Figure* is only mentioned as the context of *Artists*. I would like a sentence that has *Figure* as the subject, like "A Figure is...". Also, I think this should come before any plot to set the scope/expectation. The plot code already has a lot of details and the user does not know what the relevance of the plot/shown code is.

Reply:
As below, readers are unlikely to hit this page with no context.  They will have seen matplotlib visualizations, even if only on the landing page.  And the title clearly indicates that this is about Figures.   We need to put a figure in context of what they have seen before so I prefer to mention that first.